wiki/Main_Page/en.md

```diff
@@ -5,21 +5,27 @@ layout: main_page
 <!-- Do not add any empty lines inside this div. -->
 
 <div class="wiki-main-page__blurb">
-Welcome to the osu! wiki, a project containing a wide range of osu! related information.
+Welcome to the osu! wiki, the open-source knowledge base containing a wide range of information related to osu!, a free-to-win rhythm game.
 </div>
 
 <div class="wiki-main-page__panels">
 <div class="wiki-main-page-panel wiki-main-page-panel--full">
 
 # Getting started
 
-[Welcome](/wiki/Welcome) • [Installation](/wiki/Installation) • [Registration](/wiki/Registration) • [Help Centre](/wiki/Help_Centre) • [FAQ](/wiki/FAQ)
+Important articles and guides which will help you dive into osu!. In case anything goes wrong or appears uncertain, make sure to visit the [Help forum](https://osu.ppy.sh/forum/5).
+
+[Rules](/wiki/Rules) • [Installation](/wiki/Installation) • [Registration](/wiki/Registration) • [Help centre](/wiki/Help_Centre)
+
+[Chat console](/wiki/Chat_Console) • [BanchoBot](/wiki/BanchoBot) • [FAQ](/wiki/FAQ) • [Guides](/wiki/Guides) • [Glossary](/wiki/Glossary)
 
 </div>
 <div class="wiki-main-page-panel">
 
 # Game client
 
+Being the guide to the world of rhythm, beats, and flashes, osu! has a lot to present. Get to know it better, from various settings, through customization capabilities, to technical details and file formats the game uses.
```

Comment:
That second sentence is a bit confusing. I'm not even sure what you're trying to say here - if you're just doing a general presentation, try this:
`Get to know the game better from its various settings and customisation capabilities to technical details and file formats it uses.`